dhis2-android-app/src/main/java/org/dhis2/mobile/io/handlers/DialogHandler.java

```diff
@@ -0,0 +1,40 @@
+package org.dhis2.mobile.io.handlers;
+
+import android.app.Activity;
+
+import org.dhis2.mobile.R;
+import org.dhis2.mobile.ui.activities.DataEntryActivity;
+import org.dhis2.mobile.ui.fragments.AggregateReportFragment;
+
+public class DialogHandler {
+
+    public static void showMessage(final String message, final Activity activity) {
```

Comment:
these two methods are very similar, can you refactor this class for saving info in variables previously and create the dialog from one place without duplicate code?